cips/cip-042.md

```diff
@@ -1,14 +1,13 @@
-| cip                | 42                                                                                                                                             |
-| ------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------- |
-| title              | Matcha Network Upgrade                                                                                                                         |
-| description        | Reference specifications included in the Matcha Network Upgrade                                                                                |
-| author             | [@evan-forbes](https://github.com/evan-forbes)                                                                                                 |
-| discussions-to     | <https://forum.celestia.org>                                                                                                                   |
-| status             | Last Call                                                                                                                                      |
-| last-call-deadline | 2025-09-16                                                                                                                                     |
-| type               | Meta                                                                                                                                           |
-| created            | 2025-07-25                                                                                                                                     |
-| requires           | [CIP-36](./cip-036.md), [CIP-37](./cip-037.md), [CIP-38](./cip-038.md), [CIP-39](./cip-039.md), [CIP-40](./cip-040.md), [CIP-41](./cip-041.md) |
+| cip            | 42                                                                                                                                             |
+| -------------- | ---------------------------------------------------------------------------------------------------------------------------------------------- |
+| title          | Matcha Network Upgrade                                                                                                                         |
+| description    | Reference specifications included in the Matcha Network Upgrade                                                                                |
+| author         | [@evan-forbes](https://github.com/evan-forbes)                                                                                                 |
+| discussions-to | <https://forum.celestia.org>                                                                                                                   |
```

Comment:
FWIW this doesn't look right. There should've been a forum post for this CIP.

Reply:
yep, missed this, @evan-forbes - do we have one?

Reply:
I couldn't find one so I just created one


```suggestion
| discussions-to | <https://forum.celestia.org/t/cip-matcha-network-upgrade/2180>                                                                                                                   |
```